script.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+// завдання 1
+
+const currentMaxValue = 4589;
+let str = currentMaxValue.toString();
+let res = [];
+res.push(str);
+
+let reverseMaxValue = +res[0].split("").reverse().join("");
```

Comment:
Все рішення можна спростити до:
```suggestion
let reverseMaxValue = Number(currentMaxValue.toString().split("").reverse().join(""));
```
Не створюючи ніяких інших змінних